src/components/rangeslider/oppaxis_attributes.js

```diff
@@ -0,0 +1,45 @@
+/**
+* Copyright 2012-2018, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = {
+    // not really a 'subplot' attribute container,
+    // but this is the flag we use to denote attributes that
+    // support yaxis, yaxis2, yaxis3, ... counters
+    _isSubplotObj: true,
+
+    rangemode: {
+        valType: 'enumerated',
+        values: ['auto', 'fixed', 'match'],
+        dflt: 'match',
+        role: 'style',
+        editType: 'calc',
```

Comment:
@alexcjohnson we should be able to change these to `editType: 'plot'`, correct?

Reply:
oh hmm... actually I don't think we can, it's like `ax.autorange` which is still `calc`, because this affects `_rangesliderAutorange` which determines whether we even bother calculating `ax._min/_max` during the `calc` step.

Reply:
Ok. I tried switching them to `'plot'`, but that didn't break any test :confused: